system/models/database/abstract_db_handler.py

```diff
@@ -1,5 +1,6 @@
 from abc import ABC, abstractmethod
 from typing import List
+from system.models.users.user import User
```

Comment:
I get an error here when tried to call the user file, but I think that OK we can fix that later